packages/perseus-editor/src/components/defining-point-settings.tsx

```diff
@@ -82,6 +82,7 @@ const DefiningPointSettings = (props: Props) => {
             <CoordinatePairInput
                 coord={coord}
                 error={!!error}
+                style={styles.spaceUnder}
```

Comment:
That's a great question! I've actually been wondering about the best practice around this myself 🤔 For me, I think it depends on how cluttered the file will feel using struts vs using styles, so I've admittedly been inconsistent with it.

Here, I put `spaceUnder` so it stays consistent with the approach for the rest of the sections within this settings panel (toggle switch, color select, etc).

I also think using the same style for all of these is good in case we ever decide to change the spacing. Then we can just change it in the style and it would change everywhere. I guess if we wanted that with strut, we could make a set "small strut" component or something with the spacing hard-coded that we could reuse everywhere to keep it consistent.